python/tvm/topi/arm_cpu/conv2d.py

```diff
@@ -698,6 +737,8 @@ def schedule_conv2d_NHWC_hybrid_TIR(sch: tvm.tir.Schedule):
         "A_padded_K",
         "A_padded_M",
         "weight_flatten",
+        "weight_padding",
+        "weight_transpose",
```

Comment:
Just to check, this "weight_transpose" block shouldn't exist if "conv2d_alter_op" was performed correctly?

Reply:
Yes, that's right.